extras/videoDrivers/WinAPI/WinAPI.h

```diff
@@ -44,6 +44,9 @@ class VideoWinAPI final : public VideoDriver
     /// Die Nachrichtenschleife.
     bool MessageLoop() override;
 
+    /// Popup Fenster
```

Comment:
Can you use English here, preferably the same wording as in `VideoInterface.h` or just remove this